README.md

```diff
@@ -14,3 +16,7 @@ Change directory to the resepctive mission (e.g. cd simple-mission) and then
     cargo test
     cargo contract test
     cargo contract build
+
+## License
+
+Copyright 2023 Intergalatic Labs, Inc. Released under the GNU General Public License, version 3.0. See `LICENSE` for further details.
```

Comment:
@secretfader 
Is this really the name of our legal entity or did we copy the from from somewhere?

Reply:
Nope, this changed a few weeks back. This is now our US business entity, which holds most copyrights.

Reply:
Is it Intergalactic or Intergalatic?

Reply:
"Intergalactic." Fixed in #23.